en/docs/learn/ai/agent/introduction-to-chat-agents.md

```diff
@@ -1,4 +1,4 @@
-# Build an AI Agent to Handle Mathematical Queries
+# Introduction to Chat Agents
```

Comment:
IMO previous title with slight modifciation is better. As it is explains what this tutorial is about.

So the suggestion is `# Build a Chat Agent to Handle Mathematical Queries`

@shafreenAnfar / @MohamedSabthar

Reply:
Yeah, but the intention of this learn page is to introduce Agents. @MohamedSabthar we changed the initial paragraph as well right?

Reply:
> Yeah, but the intention of this learn page is to introduce Agents. @MohamedSabthar we changed the initial paragraph as well right?

yes